### PR TITLE
data:ssbarnea

### PR DESCRIPTION
**Pull request by ssbarnea**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3497"></a>

### CODE REVIEW

**Comment by Bereky** [comment]
Looks good!

**Review by EmmaDawsonDev — CHANGES_REQUESTED** [review]
Please add `displayStatsPublic` with a value of true or false then this will be good to go.

**Review by Pradumnasaraf — DISMISSED** [review]
Looks good to merge. Thank you for your contribution! :tada:

<p align="center"></p>

**Comment by Pradumnasaraf** [comment]
> Please add `displayStatsPublic` with a value of true or false then this will be good to go.

Hey, Emma. It's by default `false`. So, I don't think it's necessary to add that.

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Comment by ssbarnea** [comment]
@EmmaDawsonDev With lack of json json schema is quite hard to guess which fields are available.

File: data/ssbarnea.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        "icon": "FaDiscord"
+      },
+      {
+        "name": "Follow me on Facebook",
+        "url": "https://www.facebook.com/sorin.sbarnea/",
+        "icon": "FaFacebook"
+      },
+      {
+        "name": "Check me out on YouTube",
+        "url": "https://www.youtube.com/ssbarnea",
```
Comment (kumarsonsoff3) [inline data/ssbarnea.json:53]: Please fix this invalid URL, there is a missing `@`.
```suggestion
        "url": "https://www.youtube.com/@ssbarnea",
```

File: data/ssbarnea.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        "icon": "FaYoutube"
+      },
+      {
+        "name": "Check me out on Reddit",
+        "url": "https://www.reddit.com/user/sbarnea",
+        "icon": "FaReddit"
+      },
```
Comment (kumarsonsoff3) [inline data/ssbarnea.json:60]: Please remove the trailing comma `,` here to validate the `json` check.

```suggestion
      }
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Last few changes required, then it should be good to goo! 🛠

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge.

**Pull request merged**